### PR TITLE
[1LP][WIP] VM/Instance Remote Console Launch

### PR DESCRIPTION
**Pull request by mshriver**
Purpose or Intent
=================

Automate remote console launch in common.vm.

This PR is just submitting the method to open a VM console using the toolbar. control of the opened console isn't working at this point.

RHCFQE-1430

# PRT 
Doesn't impact any tests, irrelevant.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 0bfb30bddb1bfef7d5e04c2fee60415e51f99953:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3b82a4310c0165413d0a988dcde3c2ddfa36d061:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 97dbc82de865a41356293299e02c4f973cb2789d:

`cfme/common/vm.py`:
- :red_circle: Line 536:9: **F841** *local variable 'form' is assigned to but never used*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3b8f0d20b8a8e2e3e845420725fed58dbcac8248:

`cfme/common/vm.py`:
- :red_circle: Line 542:9: **F841** *local variable 'form' is assigned to but never used*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/common/vm.py [hunk 1]
```diff
@@ -525,6 +526,17 @@ def unset_ownership(self):
             action=form_buttons.save)
         flash.assert_success_message('Ownership saved for selected {}'.format(self.VM_TYPE))
 
+    def open_console(self, console='Web Console', invokes_alert=False, cancel=False):
+        # TODO: implement vmrc vm console
+        if console in ['VM Console']:
+            raise NotImplementedError('VM Console not supported at this time')
```
Comment (mshriver) [inline cfme/common/vm.py]: I should parse `console` into the exception here.

File: cfme/common/vm.py [hunk 2]
```diff
@@ -525,6 +526,17 @@ def unset_ownership(self):
...
+
+        navigate_to(self, 'Details')
+        # Click console button given by type
+        acc_btn(console, invokes_alert=invokes_alert)
```
Comment (izapolsk) [inline cfme/common/vm.py]: which alert do you expect there ?

File: cfme/common/vm.py [hunk 3]
```diff
@@ -525,6 +526,17 @@ def unset_ownership(self):
             action=form_buttons.save)
         flash.assert_success_message('Ownership saved for selected {}'.format(self.VM_TYPE))
 
+    def open_console(self, console='Web Console', invokes_alert=False, cancel=False):
```
Comment (izapolsk) [inline cfme/common/vm.py]: I dought this should be so easy. Shouldn't it return Console object or view ?

Comment (mshriver) [inline cfme/common/vm.py]: You're absolutely correct - console interaction doesn't work, this just opens the console. Still working on console view/control, wanted to merge this in so it didn't sit around and get stale. Thoughts? Maybe not worth merging separately?

File: cfme/common/vm.py (hunk 2 above)
Comment (mshriver) [inline cfme/common/vm.py]: Some of the console types have an alert, Web Console does not so its False by default.

File: cfme/common/vm.py (hunk 3 above)
Comment (psav) [inline cfme/common/vm.py]: I think we want to have something that works to merge this

Comment (mshriver) [inline cfme/common/vm.py]: Noted, works for me, I can shelve this. Thanks guys.

**Comment by mshriver** [comment]
Closing in favor of waiting for

**Pull request closed without merging**